### DISCUSSION TITLE
Multi-Collapse not working for me

### DISCUSSION THREAD

**Discussion by StokleyO** [twbs/bootstrap#39419] [Q&A] [answered]
Hey everyone! I've been trying to utilize the multi-collapse on my navbar. Basically, I couldn't house everything on my nav-bar under one collapse div, because I needed to utilize col spacing on everything. Can someone take a look and see what looks wrong about this?

```html
<nav class="navbar navbar-expand-lg mb-4" style="background-color: #5300a7;">
  <div class="container-fluid row">
    <a class="navbar-brand col-3" href="/">
      <img class="ms-4" src={logo} alt="Logo" width="125">
    </a>
    <button class="custom-toggler navbar-toggler col-2" type="button" data-bs-toggle="collapse" data-bs-target=".multi-collapse" aria-expanded="false" aria-controls="multiCollapse1 multiCollapse2" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>
    <div class="col-lg-6 d-flex justify-content-sm-start justify-content-lg-evenly navbar-nav collapse multi-collapse navbar-collapse" id="multiCollapse1">
      <a class="nav-link" href="#">First</a>
      <a class="nav-link" href="#">Second</a>
      <a class="nav-link" href="#">Third</a>
      <a class="nav-link" href="#>Fourth</a>
      <a class="nav-link" href="#">Fifth</a>
      <a class="nav-link" href="#">Sixth</a>
    </div>
    <div class="col-lg-3 d-flex justify-content-sm-evenly justify-content-lg-end collapse multi-collapse navbar-collapse" id="multiCollapse2">
      {#if userSignInStatus === false}
      <CreateAccountButton />
      <SignInButton />
      {:else}
      <div class="d-flex flex-row">
      <a href={"/user/"+userUsername} class="noStyleLinkWhite"><p class="me-4 profileNameAlign">{userUsername}</p></a>
      <SignOutButton />
      </div>
      {/if}
    </div>
  </div>
</nav>
```

**Comment by julien-deramond** [comment, marked as answer]
Hey @StokleyO 
Tried something quickly based on your example in https://codepen.io/julien-deramond/pen/xxMYgNb.
Basically I removed the `.d-flex` from the `.multi-collapse` `<div>`s and it seems to work:
- When you click to "Toggle both elements" button at desktop breakpoint, the blocks from the doc are shown but not what's in the navbar
- When you click to the toggle menu button at smaller breakpoint, both the blocks from the doc and the `.multi-collapse` `<div>`s from your example are shown

The rule `.collapse:not(.show)` in Bootstrap is equal to `display: none` so having `.d-flex` forces a display that makes them always visible.

**Reply by StokleyO** [reply]
@julien-deramond Thank you! You're a literal Godsend. That makes so much work. Is there anyway that I can repay you for this? This was driving me crazy

**Reply by julien-deramond** [reply]
No problem, glad to help :) We're all learning!

**Reply by StokleyO** [reply]
@julien-deramond Thank you so much! Bootstrap is amazing btw

**Reply by Jreleinv007** [reply]
> @julien-deramond Thank you! You're a literal Godsend. That makes so much work. Is there anyway that I can repay you for this? This was driving me crazy
